src/collections/_documentation/serverless/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Serverless'
```

Comment:
We do not have a similar top-level page for other kinds of platforms. If we think serverless integrations deserve this kind of prominence I would say this should become a marketing page in the style of https://sentry.io/for/gaming/ rather than a documentation page.

Reply:
I agree. The page content is a helpful overview (similar to what's been added for Performance), but isn't this another type of platform? So, this intro page should be nested alongside the other platforms. The files are maintained here:
src/collections/_documentation/platforms/<platform-name>/index.md

Reply:
I do think that we should have a standalone serverless page. I can do what Fiona is suggesting but don't think we should get rid of the page. @untitaker thoughts?

Reply:
If we really want this type of content in the docs (which I really don't agree with -- Performance introduces an entirely new product to the user, with SDK APIs documented which are documented nowhere else) I would like to get rid of the mention of Additional Data there. First the example is incomplete, second it's duplicating existing docs.

Reply:
Suggestion taken.